### PR TITLE
first draft of database ddl for collecting models

### PR DESCRIPTION
**Pull request by yarkivaev**

### CODE REVIEW

**Comment by evermake** [comment]
What are the advantages of creating the database comments?

File: migrations/datastore/basic_models.sql [hunk 1]
```diff
@@ -0,0 +1,171 @@
+create table contact (
+    id bigint primary key,
+    name varchar not null,
+    telegram_id bigint unique,
+    telegram_alias varchar unique,
+    phone varchar(11) unique,
+    email varchar unique,
+    extra varchar,
+    constraint contact_phone_number_format check (phone ~ '^\+?[0-9]{1,3}-?[0-9]{3}-?[0-9]{3}-?[0-9]{4}$')
```
Comment (evermake) [inline migrations/datastore/basic_models.sql:9]: 1. Regex and `phone` column length are mismatched, I think.
2. Will we store only Russian phone numbers? If so I think we should make stored format shorter and more strict.

File: migrations/datastore/basic_models.sql [hunk 2]
```diff
@@ -0,0 +1,171 @@
+create table contact (
+    id bigint primary key,
```
Comment (evermake) [inline migrations/datastore/basic_models.sql]: We should use `bigint GENERATED ALWAYS AS IDENTITY` for IDs (if we are going to use Postgres)

**Comment by yarkivaev** [comment]
> What are the advantages of creating the database comments?

Easier to understand meaning behind the models in ide like Intellj IDEA and others

**Review by evermake — CHANGES_REQUESTED** [review]

**Comment by evermake** [comment]
PR conflicts with the project structure. I've created a new PR (see #3).

**Pull request closed without merging**